app/src/main/kotlin/com/mapbox/vision/teaser/view/BaseTeaserActivity.kt

```diff
@@ -115,7 +114,10 @@ abstract class BaseTeaserActivity : BaseVisionActivity() {
         override fun onUpdateCompleted() {
             runOnUiThread {
                 if (visionManagerWasInit) {
-                    fps_performance_view.showInfo(getFrameStatistics())
+                    val frameStatistics = getFrameStatistics()
+                    if (frameStatistics != null) {
+                        fps_performance_view.showInfo(frameStatistics)
+                    }
```

Comment:
```suggestion
                    getFrameStatistics()?.let {
                        fps_performance_view.showInfo(it)
                    }
```

Reply:
It is merged to single MainActivity in latest code, let's modify in next PR,
I've made a note to fix